src/Controller/AuthenticationController.php

```diff
@@ -55,7 +55,7 @@ public function redirectAtLogin(Request $request, array $requestAttributes): Res
             $body = Stream::createFromString('');
             $response = new Response($body, '1.1', 301, '');
             /** @var Response $redirect */
-            $redirect = $response->withHeader('Location', 'http://local.quiz.com/candidate/homepage');
+            $redirect = $response->withHeader('Location', 'http://local.quiz.com/candidate/homepage?page=1');
```

Comment:
Please make a service/method in abstract controller that deals with redirects.
Also, generate your URLs from requests, hardcoded URLs are good for local testing but when you will switch environments, nothing will work anymore :)

Reply:
@andreiuta97 did you move this to another issue or are you going to do it here?

Reply:
I'm going to make a separate issue for this one if it's ok with you.